fetchApi.tsx

```diff
@@ -0,0 +1,25 @@
+import { Vehicle } from "./models";
+
+const fetchAPI = {
+  getVehiclesList: async function (page: number): Promise<[]> {
```

Comment:
`Promise<[]>` to jednak takie trochę oszukiwanie w TS - funkcja zwraca tablicę czego?
`Promise<Vehicle[]>` lub lepiej -> `Promise<ReadonlyArray<Vehicle>`